langchain/src/chains/openai_functions/base.ts

```diff
@@ -216,7 +210,9 @@ export function createStructuredOutputRunnable<
   RunOutput extends Record<string, any> = Record<string, any>
 >(
   config: CreateStructuredOutputRunnableConfig<RunInput, RunOutput>
-): RunnableInterface<RunInput, RunOutput> {
+): Runnable<RunInput, RunOutput> & {
+  pipe: Runnable<RunInput, RunOutput>["pipe"];
```

Comment:
Was changing this to just `Runnable` not enough?

Reply:
It was, shouldn't been removed before.